src/app.js

```diff
@@ -29,7 +67,23 @@ require('./middlewares/passport-auth');
 app.use(passport.initialize());
 
 app.use('/api', routes);
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.use(passport.session());
+
+app.get('/auth/twitter', passport.authenticate('twitter'));
 
+app.get(
+  '/auth/twitter/callback',
+  passport.authenticate('twitter', { failureRedirect: '/login' }),
+  (req, res) => {
+    // Successful authentication, redirect home.
```

Comment:
take google auth as an example, generate new jwt token, pass it through cookie, and send 200 response.